src/chartserver/handler_utility.go

```diff
@@ -15,6 +15,21 @@ const (
 	maxDeletionThreads = 10
 )
 
+// GetCountOfCharts calculates and returns the total count of charts under the specified namespaces.
+// See @ServiceHandler.GetCountOfCharts
+func (c *Controller) GetCountOfCharts(namespaces []string) (uint64, error) {
```

Comment:
What's the point to return the sum of chart counts under multiple namespaces?

Reply:
For the overall statistics like repo count in the project main page.